ui/values/localizable/en.go

```diff
@@ -909,12 +909,13 @@ const EN = `
 "iWillGive" = "I'll give"
 "noSupportedMarketMsg" = "Oops, there are no supported market at the moment. Please choose another server."
 "postBondMsg" = "You have zero effective tiers. Please post bonds to trade."
-"bondPostingInProgressMsg" = "You have zero effective tiers but bond posting is in progress. Please ensure your %s(%s) wallet account %s is funded with at least %s and is syncing."
+"bondPostingInProgressMsg" = "You have zero effective tiers but bond posting is in progress. Please ensure you have more than %s in the %q account of your %s (%s) wallet."
 "maxDEX" = "Max %s: %s"
 "availableBalance" = "Available Balance: %s"
 "missingDEXWalletMsg" = "No %s trade wallet found. Please add %s wallet to the DEX client to trade." 
 "swapAndRedeemFee" = "swap - %s, redeem - %s"
-"loginWithDEXPassword" = "Login with your DEX password."
+"loginWithDEXPassword" = "Login with your DEX password to trade on DCRDEX."
+"loginDEXForPendingBonds" = "You have pending bond transaction(s). Login now else your bond transactions will wait until you do!"
```

Comment:
We require login for pending bonds now?

Reply:
Yh, if the user cancelled login on app start.